lib/std/fmt.zig

```diff
@@ -467,6 +467,28 @@ pub fn formatType(
             });
             return formatType(@as(Slice, &value), fmt, options, context, Errors, output, max_depth);
         },
+        .Vector => {
+            const len = @typeInfo(T).Vector.len; // TODO .len for vectors
+            try output(context, "{ ");
+            if (len <= 16) {
```

Comment:
What's the idea behind this `len <= 16`?

Reply:
I don't know the best value.
What if vector's len == 1000? Or 10000?

Reply:
Why not remove it entirely and have control flow always go to the code you have in the `else` ?

Reply:
Ok.
I think that inlined loop is slightly faster.

Reply:
If you want to go down that route then the code should have comments to explain how the performance was measured, along with timings for one way and the simpler way so that a future maintainer can repeat the experiment.